rules/windows/persistence_evasion_hidden_local_account_creation.toml

```
@@ -15,6 +15,32 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Creation of a Hidden Local User Account"
+note = """## Triage and analysis
+
+### Investigating Creation of a Hidden Local User Account
+
+Attackers can create accounts ending with a `$` symbol to make the account hidden to user enumeration utilities and
+bypass detections that identify computer accounts by this pattern to apply filters.
```

```suggestion
bypass detections that identify suspicious computer accounts.
```
I'm not sure what you meant about applying filters. Reject this if it doesn't mean what you wanted to say.

---

Filters to avoid false positives, in this context, the attacker uses the hidden account ending with $, which can be filtered out from alerts